source/validations.ts

```diff
@@ -142,6 +147,23 @@ const validations = {
 		}
 	},
 
+	/**
+	 * Ensures a single store instance is not used with multiple express-rate-limit instances
+	 */
+	storePerLimiter(store: Store) {
```

Comment:
I think a more appropriate name (in accordance with the naming of the rest of the validations) might be `singleStorePerLimiter`. What do you think?

Reply:
Hum, I think `singleStorePerLimiter` implies that it's protecting against multiple stores being passed to a single limiter, when it's really the other way around. Maybe `singleLimiterPerStore`, or just `limiterPerStore`. Or maybe `uniqueStore` or `ownStore`?

Reply:
`freshStore` or `freshStoreInstance`?

Reply:
Some other ideas:
* `separateStore`
* `dedicatedStore`
* `unsharedStore`

Reply:
`unsharedStore` sounds good!